recipe/meta.yaml

```diff
@@ -21,8 +21,8 @@ source:
   - url: https://github.com/openvinotoolkit/mlas/archive/d1bc25ec4660cddd87804fcf03b2411b5dfb2e94.tar.gz
     sha256: 0a44fbfd4b13e8609d66ddac4b11a27c90c1074cde5244c91ad197901666004c
     folder: src/plugins/intel_cpu/thirdparty/mlas
-  - url: https://github.com/oneapi-src/oneDNN/archive/1722066ad4c0f15495f2d0fcbe9deb2bfd188c36.tar.gz  # [win64 or linux64]
-    sha256: 8955f19800066d8a705df2eb8b462085d6142b4f771ff88ae196e59a0d4ab1bb  # [win64 or linux64]
+  - url: https://github.com/oneapi-src/oneDNN/archive/0f269193c7466313888d3338209d0d06a22cc6fa.tar.gz  # [win64 or linux64]
+    sha256: abad1ff4ac138c593b7a927ef2099b01447af1f7364848392a950ba17b32bcd8  # [win64 or linux64]
```

Comment:
my understanding was that there were were some efforts to unvendor some packages such as level-zero

could we make the attempt to address this?

Could we backport https://github.com/oneapi-src/level-zero/pull/243 (early) this fix to https://github.com/conda-forge/level-zero-feedstock

and then try to use system level-zero

Reply:
We will do it as separate PR, top priority - to have 24.6

Reply:
it is ok to not way for configuration PRs to be merged in. We are VERY ok mergining these at conda-forge while upstream goes through their review process.

Oneapi is an intel project, with intel specific considerations. and we must respect those.

In the same way, you should also follow conda-forge's best practices when releasing with conda-forge infrastructure.

You asked a while back about the slow compilation times for windows.

This is in part because packages keep versionning software, and repeating the compilation process.

Unvendoring, is our way of dealing with this time complexity.

Please consider accelerating the patch to oneapi within conda-forge.

Reply:
Your understanding of processes if big companies is simplified. 

The fact that 2 components are created inside Intel does not mean anything.

Reply:
It’s not. I don’t ignore that the projects may work independently. 

But waiting for a big company to review a small PR is not something I want to do indefinitely. 

That work you did looks really high quality. 

That group at intel has many reasons to take their time with a review. 

We have a lot to gain here in making an effort to propose it to the other maintainers at conda forge.

Reply:
PR for level-zero https://github.com/conda-forge/level-zero-feedstock/pull/45